knative-operator/pkg/controller/knativeserving/kourier/kourier.go

```diff
@@ -124,3 +124,17 @@ func replaceImageFromEnvironment(prefix string, scheme *runtime.Scheme) mf.Trans
 		return nil
 	}
 }
+
+// Resources returns unstructured resources from Kourier manifest.
+func Resources(namespace string, apiclient client.Client) ([]unstructured.Unstructured, error) {
```

Comment:
Since we need to touch this again anyway (see above), what about this:

Shall we create a `func Manifest(namespace string, apiclient client.Client, scheme *scheme.Scheme) (mf.Manifest, error)` that just returns the full transformed manifest? We can always call "Resources()" on that, but more importantly, we can reuse that function in this file itself to reduce the redundancy of us loading + transforming the manifest with different transformers.

WDYT? We can do this in an iteration if you want.

Reply:
Yes, I agree. That is better.